### PR TITLE
release: use resolved nightly versions for release jobs

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
formatting bug fixed in https://github.com/openshift/ci-tools/pull/843

File: ci-operator/config/openshift/release/openshift-release-master__ocp-4.5.yaml [hunk 1]
```diff
@@ -9,26 +9,12 @@ resources:
...
-tag_specification:
-  cluster: https://api.ci.openshift.org
-  name: "4.5"
-  namespace: ocp
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.4"
```
Comment (jcpowermac) [inline ci-operator/config/openshift/release/openshift-release-master__ocp-4.5.yaml:17]: @stevekuznetsov should that be `4.5`?

**Comment by stevekuznetsov** [comment]
So unfortunately we don't have rehearsals for these periodics so to some extent this will be a blind merge. 

/retest

**Pull request merged**